packages/client/src/v2-events/features/events/useEventConfiguration.ts

```diff
@@ -20,8 +20,10 @@ export function useEventConfigurations() {
   const trpc = useTRPC()
   const config = useSuspenseQuery({
     ...trpc.event.config.get.queryOptions(),
-    queryKey: trpc.event.config.get.queryKey()
+    queryKey: trpc.event.config.get.queryKey(),
+    networkMode: 'offlineFirst'
```

Comment:
This allows React query to supply the data from cache and after that start loading